lib/web_ui/lib/src/engine/text/line_breaker.dart

```diff
@@ -25,15 +40,109 @@ enum LineBreakType {
 }
 
 /// Splits [text] into fragments based on line breaks.
-class LineBreakFragmenter extends TextFragmenter {
-  const LineBreakFragmenter(super.text);
+abstract class LineBreakFragmenter extends TextFragmenter {
+  factory LineBreakFragmenter(String text) {
+    if (domWindow.Intl.v8BreakIterator != null) {
```

Comment:
Would there be any value in running tests in the non-v8BreakIterator mode in Chrome? If so, we may need a way to override this in tests.

Reply:
We running all line breaker tests with both our fragmenter and v8BreakIterator. That should be good enough IMHO.

Other text tests are testing other aspects of text layout, not line breaks specifically. So I don't think there's value in running non-v8BreakIterator mode in Chrome. We could always change this in the future of course.